client/components/NewBankForm.js

```diff
@@ -112,19 +129,38 @@ export default class NewBankForm extends React.Component {
                 <option key={bank.id} value={bank.uuid}>{bank.name}</option>
             );
 
-            var maybeWebsites;
-            if (this.state.hasWebsites) {
-                var websitesOptions = this.state.websites.map((website) =>
-                    <option key={website.hostname} value={website.hostname}>{website.label}</option>
-                );
-                maybeWebsites = <div className="form-group">
-                    <label htmlFor="website"><T k='settings.website'>Website</T></label>
-                    <select className="form-control" id="website" ref="website">
-                        {websitesOptions}
-                    </select>
-                </div>;
+            var maybeCustomFields= [];
+            if (this.state.hasCustomFields) {
+
+                maybeCustomFields = this.state.customFields.map(function(field, index) {
+                    var customFieldFormInput;
+                    var customFieldFormInputId = "customField" + index;
+
+                    switch (field.type)
+                    {
+                        case "select":
+                            var customFieldOptions = field.values.map((opt) =>
+                                <option key={opt.value} value={opt.value}>{opt.label}</option>
+                            );
+                            customFieldFormInput = <select name={field.name} className="form-control" id={customFieldFormInputId} ref={customFieldFormInputId}>
+                                {customFieldOptions}
+                            </select>;
+                            break;
+
+                        case "text":
+                        case "number":
+                        case "password":
```

Comment:
Oh you're right: HSBC online website uses a `password` field. Can you keep the `password` case here, and change the `type` in `banks-all.json`, please?